docs/addons/writing-addons.md

```diff
@@ -109,7 +109,8 @@ Going through the code blocks in sequence:
 ```ts
 import { useGlobals, useStorybookApi } from '@storybook/manager-api';
 
-import { Icons, IconButton } from '@storybook/components';
+import { IconButton } from '@storybook/components';
+import { Icon } from '@storybook/components/experimental';
```

Comment:
do we want addon maintainers to do this, given that it's experimental?

Reply:
I found a new way to help accessibility in a semi automated way. I'll explore this more.